src/SWP/Bundle/CoreBundle/Widget/ContentListWidget.php

```diff
@@ -40,13 +39,12 @@ public function render()
         $templateName = $this->getModelParameter('template_name');
         $listId = (int) $this->getModelParameter('list_id');
 
-        /** @var ContentListRepositoryInterface $contentListRepository */
-        $contentListRepository = $this->getContainer()->get('swp.repository.content_list');
         /** @var ContentListInterface $contentList */
-        $contentList = $contentListRepository->findListById($listId);
+        $contentList = $this->getContainer()->get('swp.repository.content_list')->findListById($listId);
 
         return $this->renderTemplate($templateName, [
             'contentList' => $contentList,
```

Comment:
Yeah, but `contentList` is depreceated. I will remove it soon.  It will stay now to do not break existing templates and sites (give some time for Ljuba to fix it).